deployment/terraform/storage.tf

```diff
@@ -0,0 +1,30 @@
+#
+# S3 resources
+#
+data "template_file" "read_only_bucket_policy" {
```

Comment:
I believe that we have parts of this modularized in https://github.com/azavea/terraform-aws-s3-origin.